src/fragments/Graph.react.js

```diff
@@ -277,15 +346,27 @@ class PlotlyGraph extends Component {
 
         return (
             <div
-                key={id}
                 id={id}
-                ref={this.gd}
+                key={id}
                 data-dash-is-loading={
                     (loading_state && loading_state.is_loading) || undefined
                 }
                 style={style}
-                className={className}
-            />
+            >
```

Comment:
The graph is now wrapped in a `div` that applies the styling -- the graph div itself has consistent styling. This helps responsiveness to work correctly in flex flows.

Reply:
I'm still concerned about adding `js-plotly-plot` to the outer div - seems like having two nested divs with this class could lead to strange behavior, if it's actually getting used for something in CSS. Can you explain the reasoning here?

Reply:
Understandable. My goal here is for old versions of DDK to continue having the same behavior. Given the way DDK styling works and the stated goal of minimizing custom CSS, this should be fine. To break this behavior would require users to assign `display: flex;` to `js-plotly-plot`. The next DDK release will be using `.dash-graph`.

For the DCC world, this is in my opinion a non-issue, the general idea is that we support classes & attributes with the `dash-` prefix, everything else is implementation details.

Reply:
If we don't add `js-plotly-plot`, `dash>=1.8.0` will be a breaking version.
If we do add it, we can always create an issue to remove it and tag that issue as `Dash v2.x`.

Reply:
As pointed out by @alexcjohnson, in the end `js-plotly-plot` is not necessary as old ddk uses the forked version of the graph anyway.

The only scenario left is a new version of DDK pointing to an old version of Dash, but that can be handled through CSS with `*:not(.dash-graph) > .js-plotly-js`